README.md

```diff
@@ -280,6 +280,19 @@ master, etc. It also has variables in the `[all:vars]` section that contain sett
 useful in user playbooks. It is recommended that any user-defined Ansible playbooks should be
 managed in their own git repository (see [mikewalch/muchos-custom][mc] for an example).
 
+## High-Availability (optional)
+
+Additionally, Muchos can be configured to provide High-Availability for HDFS & Accumulo components. By default,
+this feature is off, however it can be turned on by editing the following settings in [muchos.props] 
+under the `general` section as shown below:
+
+```ini
+hdfs_ha = True                        # default is False
+nameservice_id = muchoshacluster      # Logical name for the cluster, no special characters
+```
+
```

Comment:
> > Support for optional HA was implemented keeping in mind not to disturb the existing ‘hadoop’ role but add an additional role for ‘hadoop-ha’ making both these roles independent of each other.
> 
> I am in favor of that goal. Also would not want to increase code complexity to dedupe templates. For the cases were things are significantly similar, I think it would make sense to add comments like.
> 
> ```
>   <!-- when editing this file please consider if changes are also needed in roles/hadoop-ha/templates -->
> ```

Thanks! Makes complete sense. I'll pop this comment on the config files under `hadoop`

Reply:
> Could add a sentence like the following.
> 
> ```
> Also in the `[nodes]` section of [muchos.props] ensure the `journalnode` and  `zkfc` service are configured to run.
> ```

Agreed. Thanks @keith-turner

Reply:
Addressed this on the latest commit